packages/calypso-e2e/src/lib/components/editor-popover-menu-component.ts

```diff
@@ -3,7 +3,7 @@ import { Page, Locator } from 'playwright';
 const popoverParentSelector = '.popover-slot .components-popover';
 
 const selectors = {
-	menuButton: ( name: string ) => `${ popoverParentSelector } :text-is("${ name }")`,
+	menuButton: ( name: string ) => `${ popoverParentSelector } :text("${ name }")`,
```

Comment:
@dpasque - Yes, to be honest, I was already planning on reversing this. After pushing this, I started working on some other test files, and found this change was breaking some existing tests. I've pushed a change with this. 

FYI - I had initially tried just adding :text-is("Replace header-minimal") and it didn't work. I thought it was related to the way that element is composed with the two separate items in quotes (see screenshot below). But following your comment, I tried again and it's working. I'm wondering if I left the "-" out of header-minimal, which I initially did just now. In any case, now working!

<img width="299" alt="Screenshot on 2022-10-10 at 16-58-06" src="https://user-images.githubusercontent.com/21228350/194966901-d41d70f7-033c-493c-a723-6f018f5e8819.png">

Reply:
>that element is composed with the two separate items in quotes

😱 Oh my, that does seem like that could mess that up!

And yeah, I don't love the `text-is` solution because exact matching on text nodes is miserable. 😬  But those menus are fussy enough that it seems like our best option for now. Let me know if you have any other ideas!